apps/playground-api/src/PlaygroundBlock.ts

```diff
@@ -12,7 +12,9 @@ export class PlaygroundBlock {
 
   @Interval(3000)
   async generate (): Promise<void> {
-    await this.client.call('generatetoaddress', [1, PlaygroundSetup.address, 1], 'number')
+    const { masternodes }: MiningInfo = await this.client.mining.getMiningInfo()
```

Comment:
```suggestion
    const { masternodes }: mining.MiningInfo = await this.client.mining.getMiningInfo()
```

Could we do this instead?